edx/edx-ora2
# JavaScript/SCSS make target cleanup.

**TL;DR -** `make sass` was removed a while ago without supporting doc updates. This renames `make javascript` to `make static` (to reflect the fact that it uses webpack to build both js/scss) and updates docs accordingly.

**Developer Checklist**


**Testing Instructions**

- Visual review that I haven't introduced any typos or missed instances of `make javascript`/`make sass`
- Verify I that make targets continue to work as expected

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
